openmdao/docs/features/building_blocks/components/balance_comp.rst

```diff
@@ -57,7 +57,7 @@ kwargs      dict or named arguments  Additional arguments to be passed for the c
 Example:  Scalar Root Finding
 -----------------------------
 
-The following example uses the Balance Component to implicitly solve the
+The following example uses the BalanceComp to implicitly solve the
```

Comment:
I think "a BalanceComp" sounds better than "the BalanceComp"

Reply:
I can change that, but I will point out, that the  "a vs. the" thing is not my authorship! 😃

Reply:
(I have the "a/the" change staged...will see if any other changes are needed before pushing it up.)